qatrfm/cli.py

```diff
@@ -33,7 +34,27 @@ def print_version(ctx, param, value):
     ctx.exit()
 
 
-CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
+def check_image(vars):
+    for var in vars:
+        if (var.split("=")[0] == 'image'):
```

Comment:
Why not saving the split, so we avoid it once.

Reply:
ok